packages/flutter_tools/test/integration.shard/migrate_resolve_conflicts_test.dart

```diff
@@ -0,0 +1,670 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
```

Comment:
Sorry my knowledge of the tool's null-safety migration might be a bit stale, so I don't actually know what the blockers are, but adding new non-null-safe code would seem to be taking us in the wrong direction.

Reply:
It seems that the tests are all still non-null-safe, and adding new null-safe code seems to require migrating the rest of the tests as well.